Resources/Prototypes/GameRules/events.yml

```diff
@@ -1,14 +1,14 @@
-# - type: entity
-  # id: AnomalySpawn
-  # parent: BaseGameRule
-  # noSpawn: true
-  # components:
-  # - type: StationEvent
-    # weight: 10
-    # minimumPlayers: 110
-    # startDelay: 30
-    # duration: 35
-  # - type: AnomalySpawnRule
+- type: entity
+  id: AnomalySpawn
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    weight: 10
+    minimumPlayers: 110
```

Comment:
turn the minimum players down to like 45. this event should still only be firing when there are more people around but the 110 player min means we wont ever actually see it

Reply:
> turn the minimum players down to like 45. this event should still only be firing when there are more people around but the 110 player min means we wont ever actually see it

Didnt notice it was changed, fixing it.